Overview.src.html

```diff
@@ -1315,8 +1315,20 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 <p>To perform a <dfn title=concept-fetch>fetch</dfn> using <var title>request</var>, run
 the steps below. An ongoing <span title=concept-fetch>fetch</span> can be
 <dfn title=concept-fetch-terminate>terminated</dfn> with reason <var title>reason</var>,
-which must be one of <i title>end-user abort</i>, <i title>fatal</i>, or
-<i title>timeout</i>.
+which must be one of <i title>end-user abort</i>, <i title>fatal</i>, <i title>timeout</i>, or
+<i title>garbage collection</i>.
+
+<p>The user agent may be asked to <dfn title=concept-fetch-suspend>suspend</dfn> the ongoing fetch.
+The user agent may either accept or ignore the suspension request. The susppended fetch can be
+<dfn title=concept-fetch-resume>resumed</dfn>. The user agent should ignore the suspension request
+if the ongoing fetch is updating the response in the HTTP cache for the request.
```

Comment:
I will use suspend / resume on fetch method (https://github.com/yutakahirano/fetch-with-streams/#fetch-method). I can remove them from this PR and move them to another PR if it is more preferable.